addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -59,6 +59,12 @@ const PopupWidget = publicWidget.Widget.extend({
             $(document).on('mouseleave.open_popup', () => this._showPopup());
         }
     },
+    /**
+     * @private
+     */
+    _canShowPopup() {
```

Comment:
IMO this commit should be squashed (instead of [ref] + [fix the ref] :wink:)

Reply:
Indeed, done (once pushed)